### PR TITLE
Update detect-block-potentially-unwanted-apps-microsoft-defender-anti…

### PR DESCRIPTION
**Pull request by doctordns**
…virus.md

Updated examples to have correct casing based on values in ``Microsoft.PowerShell.Cmdletization.GeneratedTypes.MpPreference.PUAProtectionType`` enum

Added an example for viewing PUA threats

Removed future tense to improve readability.

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -134,32 +134,49 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 
 See [Use PowerShell cmdlets to configure and run Microsoft Defender Antivirus](use-powershell-cmdlets-microsoft-defender-antivirus.md) and [Defender cmdlets](https://docs.microsoft.com/powershell/module/defender/index) for more information on how to use PowerShell with Microsoft Defender Antivirus.
 
 ### View PUA events
 
 PUA events are reported in the Windows Event Viewer, but not in Microsoft Endpoint Manager or in Intune.
 
+You can also use the ``Get-MpThreat`` cmdlet to view threats that Defender handled.
+```console
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:165]: Based on my personal experience with GitHub Flavored MarkDown (GFM), you don't need more than 1 pair of MD back ticks to encapsulate single-line strings, only in MD code blocks (from what I have seen so far):
```suggestion
You can also use the `Get-MpThreat` cmdlet to view threats that Defender handled.

```console
```
Note: I also recommend using the blank line as spacing between normal document text and the code block. The blank line in the code block is in practice removed in Rich Diff view and when converted to HTML on docs.microsoft.com , so we may as well remove it here.

**Comment by illfated** [comment]
Other than that, I find your changes to be very appropriate, they look good to me.

**Comment by doctordns** [comment]
Updated per comments.

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 See [Use PowerShell cmdlets to configure and run Microsoft Defender Antivirus](use-powershell-cmdlets-microsoft-defender-antivirus.md) and [Defender cmdlets](https://docs.microsoft.com/powershell/module/defender/index) for more information on how to use PowerShell with Microsoft Defender Antivirus.
 
 ### View PUA events
 
 PUA events are reported in the Windows Event Viewer, but not in Microsoft Endpoint Manager or in Intune.
 
+You can also use the `Get-MpThreat` cmdlet to view threats that Defender handled.
+
+```console
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:172]: Sorry, looks like I managed to be way too unclear in my previous suggestion. The blank line inside the code block, before the code, is basically redundant. The blank lines outside (above & below) are not.
```suggestion
```
Sorry for causing you to add unneeded spacing, that was not my intention. (I need to be more clear in my suggestion messages.)

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 3]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
 ##### To enable PUA protection
 
 ```PowerShell
-Set-MpPreference -PUAProtection enable
+
+Set-MpPreference -PUAProtection Enabled
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:139]: (revert this change)
```suggestion
Set-MpPreference -PUAProtection Enabled
```

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 4]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 ```
-Setting the value for this cmdlet to `Enabled` will turn the feature on if it has been disabled.
+Setting the value for this cmdlet to `Enabled` turns the feature on if it has been disabled.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:141]:
```suggestion
Setting the value for this cmdlet to `Enabled` turns the feature on if it has been disabled.

```

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 5]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 
 ##### To set PUA protection to audit mode
 
 ```PowerShell
-Set-MpPreference -PUAProtection auditmode
+
+Set-MpPreference -PUAProtection AuditMode
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:148]:
```suggestion
Set-MpPreference -PUAProtection AuditMode
```

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 6]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 ```
-Setting `AuditMode` will detect PUAs without blocking them.
+Setting `AuditMode` detects PUAs without blocking them.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:150]:
```suggestion

Setting `AuditMode` detects PUAs without blocking them.
```

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md [hunk 7]
```diff
@@ -134,32 +134,56 @@ For System Center 2012 Configuration Manager, see [How to Deploy Potentially Unw
...
 
 ##### To disable PUA protection
 
 We recommend keeping PUA protection turned on. However, you can turn it off by using the following cmdlet:
 
 ```PowerShell
-Set-MpPreference -PUAProtection disable
+
+Set-MpPreference -PUAProtection Disabled
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md:159]:
```suggestion
Set-MpPreference -PUAProtection Disabled
```

**Comment by illfated** [comment]
Please review & merge my PR linked above (https://github.com/doctordns/windows-itpro-docs/pull/1)

**Comment by doctordns** [comment]
looks good to me.

**Review by denisebmsft — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @doctordns

File: windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md (hunk 1 above)
Comment (denisebmsft) [reply windows/security/threat-protection/microsoft-defender-antivirus/detect-block-potentially-unwanted-apps-microsoft-defender-antivirus.md]: Just FYI, we don't say "Defender" by itself in content. We have to specify whether we mean Microsoft Defender Antivirus or Microsoft Defender for Endpoint, and so forth. I fixed this to specify Microsoft Defender Antivirus. Thank you for the contributions!

**Comment by illfated** [comment]
`*sigh*` - no good deed goes unpunished.

**Comment by illfated** [comment]
Oh well. I will make a new PR to resolve the misunderstandings from this PR.

**Comment by illfated** [comment]
PR closed and deleted.
